Updates to Google Maps API response handling and field requirements

mkokes opened the pull request:
I made some changes to how the Google Maps API response is handled and set the ability to match up multiple fields when Google Maps is set to be the geocoder.

martykokes-luminfire committed:
Updates to Google Maps API response handling and field requirements

andrewminion-luminfire reviewed (changes requested):
Could you update the readme changelog with v0.0.3 changes? Looks like you removed v0.0.2—not sure if that was intentional or not.

martykokes-luminfire committed:
No need to use + as jQuery.params() will add them for us.

martykokes-luminfire committed:
Set version back to 0.0.2

Updated readme change log and links that have since changed

martykokes-luminfire committed:
Cleaned up readme file, adding missing info back in and set version back to 0.0.3

mkokes commented:
Sorry about that I was a little confused, I added the missing info back into the readme and added 0.0.3 info to the changelog along with some url updates

andrewminion-luminfire reviewed (approved):
Without having tested it, the code changes look good. Thanks!
